### DISCUSSION TITLE
Remove failed repository from repo list

### DISCUSSION THREAD

**Discussion by myl82** [argoproj/argo-cd#8547] [Q&A] [answered]
Hello! Faced issue related of removing repository.
I've made a mistake, when adding repo URL. Repo status - Failed. I have been trying o remove it via browser ui and cli,  but without success. Repo status in cli `argocd repo list:`  
`Unable to connect to repository: rpc error: code = Unknown desc = error testing repository connectivity: Repository 'appscode https://charts.appscode.com/stable' cannot be initialized, because its root would be system temp at /tmp`

![image](https://user-images.githubusercontent.com/60928023/154427463-42e0c880-1de5-4057-82e8-aecde49b98a8.png)

Any suggestion?

**Comment by RomanBats** [comment, marked as answer]
You can try removing that [repository manifest from Secrets](https://argo-cd.readthedocs.io/en/stable/operator-manual/declarative-setup/#repositories) in Kubernetes manually.

**Reply by myl82** [reply]
Thanks! Works for me

**Reply by mbrozzo** [reply]
Heello, I have the same problem, as described in https://github.com/argoproj/argo-cd/issues/8774.
I am using the ArgoCD operator in OKD but I cannot find the repository secrets in the secrets section of OKD.

Could you please give me some clearer directions on how to find them? Thank you in advance!

**Reply by mbrozzo** [reply]
Update: I was able to solve it, see https://github.com/argoproj-labs/argocd-operator/issues/1497#issuecomment-2270775761.
